docs/configuration.rst

```diff
@@ -454,6 +454,13 @@ The following environment variables for the tracer are supported:
      default: True
      description: Send query strings in http.url tag in http server integrations.
 
+   DD_TRACE_HTTP_SERVER_ERROR_STATUSES:
+     type: String
+     default: "500-599"
+     description: |
+        Comma-separated list of HTTP status codes that should be considered errors when returned by an HTTP request.
+        The status codes are used to set the ``error`` field on the span.
```

Comment:
So comma separated and dashes? So they can mix like:
```
DD_TRACE_HTTP_SERVER_ERROR_STATUSES="200,400-404,500-599"
```
Maybe we should explicitly specify that they can also specify ranges with `-`?